### ISSUE TITLE
Can you please add the video thumbnail to the cover art of the created mp3 ?

### ISSUE THREAD

**Issue by LL25255252** [bradleyasu/ytdl50#2]
Running ver. 5.1.3
Win 7 x64

Thanks in advance.

**Comment by bradleyasu** [comment]
Yes, this should be do-able.  I'll try to get it in the upcoming update!  Thanks for the suggestion - I'll leave this issue open until I get it in.
